pandas/tests/frame/test_mutate_columns.py

```diff
@@ -195,7 +195,9 @@ def test_delitem_multiindex(self):
 
         # xref: https://github.com/pandas-dev/pandas/issues/2770
         # the 'A' is STILL in the columns!
-        assert 'A' in df.columns
+        # the above was changed, 'A' is no longer in columns
```

Comment:
you comment doesn't make sense here once the removed line is gone, can you clarify (and you can reference 2770), but make a nicer comment overall